### PR TITLE
[xcode16.1] Bump Xcode to Xcode 16.1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/scenekit.cs [hunk 1]
```diff
@@ -1967,7 +1967,7 @@ interface SCNSceneLoadingOptions {
 	[MacCatalyst (13, 1)]
 	[BaseType (typeof (NSObject))]
 	interface SCNNode : SCNAnimatable, SCNBoundingVolume, SCNActionable, NSCopying, NSSecureCoding
-#if IOS || TVOS
+#if (IOS || TVOS) && !XAMCORE_6_0 // Conformance Removed in Xcode 16.1
```
Comment (rolfbjarne) [inline src/scenekit.cs:1970]: No need to wait that long:

```suggestion
#if (IOS || TVOS) && !XAMCORE_5_0 // Conformance Removed in Xcode 16.1
```

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 2]
```diff
@@ -975,7 +975,7 @@ protected override bool Skip (Type type, string protocolName)
 					return !TestRuntime.CheckXcodeVersion (8, 0);
 				case "SCNNode":
 				case "SCNReferenceNode":
-					return !TestRuntime.CheckXcodeVersion (9, 0);
+					return TestRuntime.CheckXcodeVersion (16, 1); // Conformance removed in Xcode 16.1 will be removed in XAMCORE_6_0
```
Comment (rolfbjarne) [inline tests/introspection/iOS/iOSApiProtocolTest.cs:978]:
```suggestion
					return TestRuntime.CheckXcodeVersion (16, 1); // Conformance removed in Xcode 16.1 will be removed in XAMCORE_5_0
```

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 3]
```diff
@@ -1095,7 +1095,7 @@ protected override bool Skip (Type type, string protocolName)
...
```
Comment (rolfbjarne) [inline tests/introspection/iOS/iOSApiProtocolTest.cs:1098]:
```suggestion
					return TestRuntime.CheckXcodeVersion (16, 1); // Conformance removed in Xcode 16.1 will be removed in XAMCORE_5_0
```

File: src/coreimage.cs [hunk 4]
```diff
@@ -10274,6 +10274,18 @@ interface CIAreaBoundsRed : CIAreaBoundsRedProtocol {
...
 	interface CIAreaBoundsRedProtocol : CIAreaReductionFilterProtocol {
 	}
+
+	[CoreImageFilter]
+	[TV (18, 1), Mac (15, 1), iOS (18, 1), MacCatalyst (18, 1)]
+	[BaseType (typeof (CIFilter))]
+	interface CIDistanceGradientFromRedMask : CIFilterProtocol {
+
+		[CoreImageFilterProperty ("inputImage")]
+		CIImage inputImage { get; set; }
```
Comment (rolfbjarne) [inline src/coreimage.cs:10284]:
```suggestion
		CIImage InputImage { get; set; }
```

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Comment by dalexsoto** [comment]
known issue https://github.com/xamarin/xamarin-macios/issues/20117

**Pull request merged**